firmware/application/tone_key.hpp

```diff
@@ -30,6 +30,9 @@ using namespace ui;
 
 namespace tonekey {
 
+#define TONE_FREQ_TOLERANCE_HZ 4.0
+#define TONE_DISPLAY_TOGGLE_COUNTER 2
+
 typedef int32_t tone_index;
 
 using tone_key_t = std::vector<std::pair<std::string, float>>;
```

Comment:
I don't know much about this area, but why are these floats? Almost nothing on the device uses floats.

Reply:
Also, there's no FPU so it might be better to rework this using ints at some point unless floats are necessary for some reason.

Reply:
Agree totally.  Especially when the baseband code is sending the frequency over as an integer!  The tone_key table used floating point values for source code readability I guess, but that is a lame reason.  I'll fix it.